src/connectors/hits-per-page/connectHitsPerPage.js

```diff
@@ -108,8 +108,13 @@ export default function connectHitsPerPage(renderFn, unmountFn = noop) {
     }
 
     const defaultItems = items.filter(item => item.default === true);
+    const defaultItemsCount = defaultItems.length;
 
-    if (defaultItems.length > 1) {
+    if (items.length && defaultItemsCount === 0) {
+      throw new Error(
+        withUsage(`A default value must be specified in \`items\`.`)
+      );
+    } else if (defaultItemsCount > 1) {
```

Comment:
```suggestion
    }
    if (defaultItemsCount > 1) {
```

throwing closes a branch